modules/mig/variables.tf

```diff
@@ -52,20 +52,32 @@ variable "distribution_policy_zones" {
   default     = []
 }
 
+#################
+# Stateful disk
+#################
+variable "stateful_disk" {
```

Comment:
Since this is a list, can we call it `stateful_disks`?

Reply:
sure, np with that.